modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl

```diff
@@ -609,11 +609,13 @@ public abstract class Base${schemaName}ResourceImpl
 		this.roleLocalService = roleLocalService;
 	}
 
-	<#if generateBatch>
+	<#if configYAML.generateBatch>
 		public void setVulcanBatchEngineImportTaskResource(VulcanBatchEngineImportTaskResource vulcanBatchEngineImportTaskResource) {
 			this.vulcanBatchEngineImportTaskResource = vulcanBatchEngineImportTaskResource;
 		}
+	</#if>
 
+	<#if generateBatch>
```

Comment:
I find weird to use different parameters for validate batch generations, but I guess in the following PRs we will centralize the functionality to a method in the FreeMarkerTool

Reply:
As I was in a hurry and I want to forward this PR, so you can rebase yours and send it, I thought to do it this way and centralize in the next PR